playbooks/robusta_playbooks/events_enrichments.py

```diff
@@ -0,0 +1,56 @@
+from robusta.api import *
+
+
+class EventErrorReportParams(FindingKeyParams, RateLimitParams):
+    pass
+
+
+@action
+def event_report(event: EventChangeEvent, action_params: EventErrorReportParams):
+    """
+    Create finding based on the kubernetes event
+    """
+    k8s_obj = event.obj.involvedObject
+
+    # creating the finding before the rate limiter, to use the user the service key for rate limiting
```

Comment:
I don't understand the comment. Specificaly what does it mean "to use the user the service key for rate limiting"

Reply:
it's a typo.
it's supposed to be:
creating the finding before the rate limiter, to use the service key for rate limiting